src/stx/common/trex_latency_counters.cpp

```diff
@@ -141,6 +141,10 @@ RXLatency::handle_pkt(const rte_mbuf_t *m) {
     uint8_t tmp_buf[sizeof(struct flow_stat_payload_header)];
     CFlowStatParser parser(CFlowStatParser::FLOW_STAT_PARSER_MODE_SW);
     int ret = parser.parse(rte_pktmbuf_mtod(m, uint8_t *), m->pkt_len);
+    if ( CGlobalInfo::m_options.m_ip_cfg[0].get_vxlan_fs() ) {
+        uint16_t vxlan_skip = parser.get_vxlan_payload_offset(rte_pktmbuf_mtod(m, uint8_t *), m->pkt_len);
```

Comment:
```get_vxlan_payload_offset ``` can't be used in this location. ```get_vxlan_payload_offset ``` is a CP function that throw exceptions for RPC. 
In this case you are processing packets in the DP and it is not possible to crash. 
You need to change the parser. parses function to skip the vxlan header (if exits and enable) exactly like what the ASIC does